modules/oe_theme_content_publication/src/Plugin/ExtraField/Display/PublicationCollection.php

```diff
@@ -0,0 +1,154 @@
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * PublicationCollection constructor.
+   *
+   * @param array $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin_id for the plugin instance.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, EntityTypeManagerInterface $entity_type_manager) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->entityTypeManager = $entity_type_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('entity_type.manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getLabel() {
+    return $this->label;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(ContentEntityInterface $entity) {
+    $build = [];
+    /** @var \Drupal\node\Entity\Node[] $collections */
+    $collections = $this->getReferencingCollections($entity);
+
+    if (empty($collections)) {
+      return $build;
+    }
+
+    $build = [
+      '#theme' => 'oe_theme_content_publication_collections',
+    ];
+
+    // Set the label of the field.
+    if (count($collections) > 1) {
+      $this->label = $this->t('Part of collections');
+    }
+    else {
+      $this->label = $this->t('Part of collection');
+    }
+
+    $cache = CacheableMetadata::createFromRenderArray($build);
+
+    $items = [];
+    foreach ($collections as $collection) {
+      $cache->addCacheableDependency($collection);
+      $items[] = [
+        '#type' => 'link',
+        '#title' => $collection->label(),
+        '#url' => Url::fromRoute('entity.node.canonical', ['node' => $collection->id()]),
+        '#attributes' => [
+          'class' => ['ecl-link', 'ecl-link--standalone'],
+        ],
+      ];
+    }
+
+    $build['#items'] = $items;
+
+    $cache->applyTo($build);
+
+    return $build;
+  }
+
+  /**
+   * Get the publication collections that are referencing other publications.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The publication entity.
+   *
+   * @return array
+   *   The loaded publication entities.
+   */
+  protected function getReferencingCollections(EntityInterface $entity): array {
+    $referencing_entities = [];
+
+    // Load all entities that have a reference to the given entity.
+    $entity_type_storage = $this->entityTypeManager->getStorage('node');
+    $query = $entity_type_storage->getQuery('OR');
```

Comment:
```suggestion
    $query = $entity_type_storage->getQuery();
```
Let's make it simpler, we don't need "or" condition here.